account_banking_pain_base/models/account_config_settings.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Akretion - Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
```

Comment:
The file `account_config_settings.py` should be renamed to `res_config_settings.py`